### PR TITLE
Add required ciphersuites for connectivity to Azure Front Door

### PR DESCRIPTION
**Pull request by pshelton-skype**
Some Office 365 products (including Microsoft Teams) use Azure Front Door (AFD) to terminate TLS connections. When enforcing TLS 1.2, AFD supports a much smaller subset of ciphersuites customers need to have enabled to successfully connect. 

This commit:
- Adds a new column to the required ciphersuites table to indicate that one of the two EDCHE GCM ciphersuites is required for compatibility with AFD
- Add links back to Azure Front Door documentation of their supported ciphersuites

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/technical-reference-details-about-encryption.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/technical-reference-details-about-encryption.md]:
```suggestion
Certain Office 365 products (including Microsoft Teams) use [Azure Front Door](/azure/frontdoor/front-door-overview) to terminate TLS connections and route network traffic efficiently. One of the [cipher suites supported by Azure Front Door over TLS 1.2](/azure/frontdoor/front-door-faq#what-are-the-current-cipher-suites-supported-by-azure-front-door-) must be enabled to successfully connect to these products. We recommend enabling one of the two ECDHE cipher suites marked "Required for AFD" in the table below for maximum security. 
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @pshelton-skype, kindly commit my suggested changes. Many thanks.

**Comment by pshelton-skype** [comment]
@JohanFreelancer9 - Done. Thanks for the suggestions.

**Comment by pshelton-skype** [comment]
Got signoff from the Teams representative during 1-1 discussion. Still waiting on reviews from Office and our security champs.

**Review by JohanFreelancer9 — APPROVED** [review]

File: microsoft-365/compliance/technical-reference-details-about-encryption.md
Comment (qiufasheng) [inline microsoft-365/compliance/technical-reference-details-about-encryption.md]: Suggest indicating below 2 ciphers are supported by AFD for for Windows 7 and 8 compatibility.
TLS_DHE_RSA_WITH_AES_256_GCM_SHA384 (0x9f)   DH 2048 bits
TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 (0x9e)   DH 2048 bits

**Comment by qiufasheng** [comment]
+ @KCCross

File: microsoft-365/compliance/technical-reference-details-about-encryption.md
Comment (pshelton-skype) [reply microsoft-365/compliance/technical-reference-details-about-encryption.md]: You'd like me to make that comment in the AFD documentation? We don't have any discussion of those two cipher suites in this doc. We're documenting the non-FS `TLS_RSA` but not the `TLS_DHE_RSA`.

Comment (pshelton-skype) [reply microsoft-365/compliance/technical-reference-details-about-encryption.md]: Discussed offline with Fasheng. My mistake was thinking Win 7+ supported EDCHE. Only Win 10+ supports ECDHE and we need to clarify our recommendation that any of the four cipher suites may need to be enabled depending on the OS version.

**Comment by pshelton-skype** [comment]
@KCCross, @qiufasheng, @JohanFreelancer9 - can you please sign off on the latest version?

**Review by qiufasheng — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]
#sign-off

**Comment by KCCross** [comment]
Sign off from me too.

**Review by pshelton-skype — COMMENTED** [review]
@yogkumgit - Looks like this is ready to merge.

**Pull request merged**
